internal/service/teams/resource_teams_test.go

```diff
@@ -32,42 +32,39 @@ func TestAccConfigRSTeam_basic(t *testing.T) {
 		CheckDestroy:             acc.CheckDestroyTeam,
 		Steps: []resource.TestStep{
 			{
-				Config: testAccMongoDBAtlasTeamConfig(orgID, name,
+				Config: configBasic(orgID, name,
```

Comment:
you can change username to this and use it in all places:

username     =[]string{  os.Getenv("MONGODB_ATLAS_USERNAME") }